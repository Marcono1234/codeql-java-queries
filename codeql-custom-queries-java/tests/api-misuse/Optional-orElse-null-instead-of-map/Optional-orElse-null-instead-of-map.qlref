./api-misuse/Optional-orElse-null-instead-of-map.ql
