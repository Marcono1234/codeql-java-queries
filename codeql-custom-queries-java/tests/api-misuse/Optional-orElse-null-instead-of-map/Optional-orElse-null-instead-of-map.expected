| Test.java:8:16:8:31 | orElse(...) | Use Optional.map(...) and remove null check $@. | Test.java:9:18:9:48 | ...?...:... | here |
| Test.java:11:16:11:31 | orElse(...) | Use Optional.map(...) and remove null check $@. | Test.java:12:18:12:48 | ...?...:... | here |
